io.github.andrewmkiss.Demeter.yml

```diff
@@ -0,0 +1,190 @@
+      - '/bin/shasum'
+      - '/bin/splain'
+      - '/bin/xsubpp'
+      - '/bin/zipdetails'
+      - '/include'
+      - '/man'
+      - '*.pod'
+
+  ## Using the dependencies from flatpak-cpan-generator
+  # Installs the modules generated by flatpak-cpan-generator.
+  # Note that *any* Makefile.PL-style modules MUST be installed in this one step,
+  # as once perllocal.pod is written in one module, it cannot be modified by others.
+  - name: perl-libs
+    disabled: false
+    buildsystem: simple
+    build-commands:
+      - 'perl-libs/install.sh'
+    # Same as with the Perl module, we need to restore write permission.
+    # However, -f is now passed to avoid errors from trying to touch files from the
+    # above module that are now marked as read-only.
+    post-install:
+      - 'chmod -Rf u+w /app/lib/perl5/site_perl'
+      - 'chmod u+w /app/bin/pdl*'
+    sources:
+      - perl-libs.yml
+    # This step should be customized based on the CPAN packages you're using.
+    cleanup:
+      - '/bin'
+      - '/man'
+
+  ## Demeter
+  - name: demeter
+    disabled: false
+    buildsystem: simple
+    build-options:
+      env:
+        - PERL5LIB=/run/build/demeter:/app/lib/perl5/site_perl
+        - PERLLIB=/app/lib/perl5/site_perl/5.36.1
+    build-commands:
+      - perl Build.PL
+      - sed -i "1653,+3s/25/35/" lib/Demeter/UI/Athena.pm
+      - ./Build
+      - ./Build install
+      - install -Dm 644 metainfo.xml /app/share/metainfo/io.github.andrewmkiss.Demeter.metainfo.xml
+      - install -Dm 644 /app/lib/perl5/site_perl/5.36.1/x86_64-linux/Demeter/share/Demeter_icon.png /app/share/icons/hicolor/48x48/apps/io.github.andrewmkiss.Demeter.png
+      - install -Dm 644 /app/lib/perl5/site_perl/5.36.1/x86_64-linux/Demeter/UI/Athena/share/athena_icon.png /app/share/icons/hicolor/48x48/apps/io.github.andrewmkiss.Demeter.Athena.png
+      - install -Dm 644 /app/lib/perl5/site_perl/5.36.1/x86_64-linux/Demeter/UI/Artemis/share/artemis_icon.png /app/share/icons/hicolor/48x48/apps/io.github.andrewmkiss.Demeter.Artemis.png
+      - ffmpeg -i Demeter.png -s 128x128 -frames:v 1 Demeter_128x128.png
+      - install -Dm 644 Demeter.png /app/share/app-info/icons/flatpak/512x512/io.github.andrewmkiss.Demeter.png
+      - install -Dm 644 Demeter_128x128.png /app/share/app-info/icons/flatpak/128x128/io.github.andrewmkiss.Demeter.png
+      - install -Dm 644 -t /app/share/applications io.github.andrewmkiss.Demeter.Athena.desktop
+      - install -Dm 644 -t /app/share/applications io.github.andrewmkiss.Demeter.Artemis.desktop
+      - install -Dm 644 -t /app/share/applications io.github.andrewmkiss.Demeter.Hephaestus.desktop
+    post-install:
+      - 'chmod u+w /app/lib/perl5/site_perl/5.36.1/x86_64-linux/auto/Ifeffit/Ifeffit.so'
+    sources:
+      - type: git
+        url: https://github.com/bruceravel/demeter
+      # branch: master
+        commit: edfaa0904fa30426cf303b5933450edf22a426ec
+      - type: file
+        path: metainfo.xml
```

Comment:
this file doesn't exist. 

did you really build this locally?

Reply:
The instructions said to copy the manifest into the branch. I assumed that only meant the `yml` manifest file(s). I will add the additional files.
This does build locally.

Reply:
![image](https://github.com/flathub/flathub/assets/114441/a048b8fe-b76d-4af0-9d70-2a262b26400a)

file doesn't exist